src/gatsby/src/styles/parks.scss

```diff
@@ -481,9 +481,8 @@ h2.section-heading {
   margin-bottom: 16px;
   &--left {
     svg {
-      width: 36px;
-      height: 36px;
-      color: $colorBlue;
+      width: 48px;
+      height: 48px;
```

Comment:
If you ever need to show these icons in different colors, you can use the currentColor CSS property to behave like the web font and inherit the color of the `span` tag around it 👍